draft-ietf-jsonpath-base.md

```diff
@@ -1104,6 +1094,56 @@ Some examples:
 | `[3,4,5]` | `$[?index(@)==2]`<br>`$[?index(@)==17]` | `[5]`<br>`[]` | Select array element |
 | `{"a":{"b":{5},c:0}}` | `$[?@.b==5 && !@.c]` | `[{"b":{5},c:0}]` | Existence  |
 
+
+## Normalized Paths
+
+A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node. For example,
+the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
+`$['book'][3]` and `$['book'][5]`. For a given JSON document, there is a one to one correspondence between the document's
+nodes and the Normalized Paths that identify these nodes.
+
+A JSONPath implementation may output Normalized Paths instead of, or in addition to, the values identified by these paths.
+
+Since bracket notation is more general than dot notation, it is used to construct Normalized Paths.
+Single quotes are used to delimit string member names. This reduces the number of characters that
+need escaping when Normalized Paths appear as strings (which are delimited with double quotes) in JSON documents.
+
+The syntax of Normalized Paths is restricted so that there is one and only one way of representing any
+given Normalized Path. Putting this another way, for any two distinct Normalized Paths, there is a JSON document
```

Comment:
```suggestion
given Normalized Path. Putting this another way, for any two distinct Normalized Paths, a JSON document exists such that
it will yield distinct results when the Normalized Paths are applied to it.
```

Reply:
I pushed a commit with a variation of that suggestion.